django/forms/templates/django/forms/widgets/select_option.html

```diff
@@ -1 +1 @@
-<option value="{{ widget.value }}"{% include "django/forms/widgets/attrs.html" %}>{{ widget.label }}</option>
+<option value="{{ widget.value|stringformat:'s' }}" {% include "django/forms/widgets/attrs.html" %}>{{ widget.label }}</option>
```

Comment:
Any reason for the space between the `value` ending `"` and the `{% include %}` tag? It looks like `attrs.html` already takes care of adding it.